development/scripts/fix-spec.py

```diff
@@ -0,0 +1,107 @@
+                if 'properties' in type_property and 'value' in type_property['properties']:
+                    value_property = type_property['properties']['value']
+                    if 'enum' in value_property and set(value_property['enum']) == {'primary', 'redundant'}:
+                        print(f"Replacing complex 'type' field in PowerFeed")
+                        schema['properties']['type'] = {
+                            'type': 'string',
+                            'enum': ['primary', 'redundant'],
+                            'default': 'primary'
+                        }
+
+            # Replace nested `supply` field in PowerFeed
+            if 'supply' in schema['properties']:
+                supply_property = schema['properties']['supply']
+                if 'properties' in supply_property and 'value' in supply_property['properties']:
+                    value_property = supply_property['properties']['value']
+                    if 'enum' in value_property and set(value_property['enum']) == {'ac', 'dc'}:
+                        print(f"Replacing complex 'supply' field in PowerFeed")
+                        schema['properties']['supply'] = {
+                            'type': 'string',
+                            'enum': ['ac', 'dc'],
+                            'default': 'ac'
+                        }
+
+            # Replace nested `phase` field in PowerFeed
+            if 'phase' in schema['properties']:
+                phase_property = schema['properties']['phase']
+                if 'properties' in phase_property and 'value' in phase_property['properties']:
+                    value_property = phase_property['properties']['value']
+                    if 'enum' in value_property and set(value_property['enum']) == {'single-phase', 'three-phase'}:
+                        print(f"Replacing complex 'phase' field in PowerFeed")
+                        schema['properties']['phase'] = {
+                            'type': 'string',
+                            'enum': ['single-phase', 'three-phase'],
+                            'default': 'single-phase'
+                        }
+        
+        # Handle Prefix schema
+        if name == 'Prefix' and 'properties' in schema:
+            # Replace complex `type` object with a simpler string enum in Prefix
+            if 'type' in schema['properties']:
+                type_property = schema['properties']['type']
+                if 'properties' in type_property and 'value' in type_property['properties']:
+                    print(f"Replacing complex 'type' field in Prefix")
+                    schema['properties']['type'] = {
+                        'type': 'string',
+                        'enum': ['container', 'network', 'pool'],
+                        'default': 'network'
+                    }
+                    
+        if 'properties' in schema:
+            # Remove "null" item from nullable enums
```

Comment:
Can you point to an example of this case? I'm not finding one in the schema at a glance.

Reply:
This was kept from the netbox script.
I know that front_image and rear_image do not like the nullable property and some other properties as well.
I'll have a look at this tomorrow and check if this is really required.

Reply:
This isn't required, so I removed this section